src/sass/base/_conteiner.scss

```diff
@@ -0,0 +1,26 @@
+.main-container {
+  margin-left: auto;
+  margin-right: auto;
+
+  @media screen and (min-width: 320px) {
```

Comment:
You should use not px when writing min-width. Add in file `utils/_breakpoints.scss` breakpoints

Reply:
Change `px` on `$mobile`